### PR TITLE
[RNMobile] Add block outline to all Social Link blocks when selected

### PR DESCRIPTION
**Pull request by derekblank**
## Related PRs

* https://github.com/WordPress/gutenberg/pull/54011 by @derekblank

## Testing Instructions
1. Create a Post and add a Social Links block.
2. Tap one of the four initial blocks (WordPress, Facebook, Twitter, Instagram) and note the blue Block Outline appears when the block is selected.
3. Tap the [+] button to the right of the initial blocks to open the Social Link block inserter. 
4. Add any block from the inserter: Amazon, Dribbble, Dropbox, etc.
5. When the new block is added, the blue Block Outline should appear around the new block. When de-selected by tapping anywhere else in the editor, and reselecting the block by tapping it, the blue Block Outline should appear.

### CODE REVIEW

**Comment by derekblank** [comment]
@dcalhoun I am unsure why wpmobilebot continues to close https://github.com/wordpress-mobile/gutenberg-mobile/pull/6120 (other than the message "This PR is closed because there is no longer an associated gutenberg PR for it."). Is it because that PR is not assigned to anyone, perhaps? Re-opening this companion PR instead.

I wanted to confirm that I am following the correct sequence of steps for updating the companion PRs when the Gutenberg Mobile PR has a conflict with Gutenberg ref. 

The steps I followed:
1. Noted that https://github.com/wordpress-mobile/gutenberg-mobile/pull/6120 and https://github.com/WordPress/gutenberg/pull/54011 both had approvals
2. Merged gutenberg#54011
3. In Gutenberg Mobile, pulled the latest from `trunk` and merged it into `version-toolkit/gutenberg/rnmobile/outline-all-social-link-block`
4. In that branch, updated the Gutenberg ref to `trunk`, which would also contain the merge of gutenberg#54011
5. Wait for CI to pass in the Gutenberg Mobile PR and then merge it.

Let me know if there's a step you would have done differently. 🙇

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@ Unreleased
 * [*] Remove third-party dependency on react-native-hsv-color-picker [https://github.com/WordPress/gutenberg/pull/53329]
 * [*] Search Control - Prevent calling TextInput's methods when undefined [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6090]
 * [**] Add basic support to view, relocate, and remove the Jetpack Paywall block. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/6076]
+* [**] [RNMobile] Add block outline to all Social Link blocks when selected [https://github.com/WordPress/gutenberg/pull/54011]
```
Comment (dcalhoun) [inline RELEASE-NOTES.txt]: IMO the `[RNMobile]` tag isn't relevant or helpful in release notes, which ultimately are user-facing. All changes in these release notes relate to the mobile editor. 
```suggestion
* [**] Add block outline to all Social Link blocks when selected [https://github.com/WordPress/gutenberg/pull/54011]
```

**Review by dcalhoun — APPROVED** [review]
> I am unsure why wpmobilebot continues to close #6120 (other than the message "This PR is closed because there is no longer an associated gutenberg PR for it."). Is it because that PR is not assigned to anyone, perhaps?

Yes. A cron job runs that will close any `gutenberg-mobile` PR whose sibling `gutenberg` PR is [closed if](https://github.com/wordpress-mobile/version-toolkit/blob/96e3812503892ed7cb81f770a73caba4bc314039/lib/submodule_update_flow.rb#L43-L54) the `gutenberg-mobile` PR does not have an assignee _and_ it only contains commits authored by `wpmobilebot`. Additional information can be found in p9ugOq-2AI-p2.

> Let me know if there's a step you would have done differently. 🙇

No, the steps you outlined are appropriate. In relation to avoiding auto-closure of the `gutenberg-mobile` PR, pushing the updated `gutenberg` ref commit in a timely fashion might've helped avoid erroneous auto-closure (not guaranteed, however, as the cron job could run at any moment). 

However, I recommend assigning yourself to all of your PRs if your intention is to merge it yourself. Doing so communicates your intent very clearly all — bots or humans. Additionally, [`WordPress-Android`](https://github.com/wordpress-mobile/WordPress-Android/blob/f5fe8614c2e935c42446db2c79d08b9780a7f995/CONTRIBUTING.md#pr-merge-policy) and [`WordPress-iOS`](https://github.com/wordpress-mobile/WordPress-iOS/blob/fa8f3afa409849948c4908e07c6192fdb45dd818/CONTRIBUTING.md#pr-merge-policy) have differing approaches to merging PRs. The former expects PR reviewers to merge work unless an assignee is set. I am unsure how often this procedure is followed, but assigning yourself helps avoid confusion in many ways.

**Comment by derekblank** [comment]
Thanks for confirming the merge process and for explaining the automated PR closure -- much appreciated. Good to know I was only missing the PR assignment step, which I agree is a good practice to follow for clarity.

**Pull request merged**